src/AppInstallerCLITests/TestData/InstallFlowTest_ExpectedReturnCodes.yaml

```diff
@@ -1,7 +1,7 @@
-PackageIdentifier: AppInstallerCliTest.TestInstaller
+PackageIdentifier: AppInstallerCliTest.ExpectedReturnCodes

```

Comment:
Is this Id and name change intentional? Will it affect what other tests were testing?

Reply:
Name change is intentional as now the id clearly identifies what the purpose of the manifest. Also helps avoid conflicts when creating the test source for update/import/dependency sources.